BUILD

```diff
@@ -73,7 +70,7 @@ genrule(
     outs = [
         "gflags_completions.h",
     ],
-    cmd = "awk '{ gsub(/@GFLAGS_NAMESPACE@/, \"gflags\"); print; }' $(<) > $(@)",
+    cmd = "awk '{ gsub(/@GFLAGS_NAMESPACE@/, \"google\"); print; }' $(<) > $(@)",
```

Comment:
That's a difficult one. Currently the main namespace is `google` to maintain ABI compatibility with gflags version 2.0. However, the next major release will switch to `gflags` to emphasise that this version of the gflags library may not be the same as Google's internal version.

This may break the build of projects of Bazel based projects that already depend on this external package, in which case they will have used the new `gflags` namespace in their source code.